packages/fast-form-generator-react/src/form/form.utilities.ts

```diff
@@ -1,5 +1,23 @@
 import { clone, cloneDeep, get, isEqual, mergeWith } from "lodash-es";
-import { IBreadcrumbItemConfig, IBreadcrumbItemsConfig, IComponentItem, IFormProps, IFormState, LocationOnChange } from "./form.props";
+import * as tv4 from "tv4";
+import { IBreadcrumbItemConfig, IBreadcrumbItemsConfig, IComponentItem, IFormProps, IFormState } from "./form.props";
+
+export enum RegexType {
```

Comment:
I think this is more ceremony than is really necessary - I really just meant:
```
const squareBrackets: RegExp = /\[(\d+?)\]/g;
const oneOfAnyOf: RegExp = /(oneOf|anyOf)\[\d+\]/g;
```

I'd prefer that approach because it doesn't require us to semantically name and export both an enum and a function to retrieve from that enum